### DISCUSSION TITLE
Hey, is there a way to "unbrick" the school chromebook?

### DISCUSSION THREAD

**Discussion by GGboogie** [3kh0/ext-remover#620] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I ask this because on another chromebook that I had was not controlled or was not managed by the school admin and so, there was no goguardian extension or any extensions from the school that would have blocked the chromebook. I'm still not sure why the chromebook wasn't administered but I don't have the chromebook anymore. I also got the chromebook from the school's library where you would usually ask the librarian for another school chromebook if you had lost your previous one or it was broke. So I ask how do you unbrick the chromebook? Because even when you do "esc + reload tab button + power button" it does absolutely nothing. Also doing "ctrl + shift + alt + r" it also does nothing. So please, if anyone has a way to do so, please reply.

**Comment by BroJac5246** [comment]
What did you do to brick it? Were you trying SH1mmer?

**Comment by funkymokey** [comment]
it dosent turn on?

**Comment by GGboogie** [comment]
It's blocked by cisco umbrella and goguardian and it can't powerwash and or developer mode won't turn on.

**Reply by BroJac5246** [reply]
What did you do to it, though? We can't help you if we don't know.

**Comment by jessiejs** [comment, marked as answer]
do hold escape and refresh, then press power
if that doesn't work charge the device
if that doesn't work cry.

**Reply by Hilololsusy** [reply]
POV: you read the post

**Comment by Aka-but-nice** [comment]
> I ask this because on another chromebook that I had was not controlled or was not managed by the school admin and so, there was no goguardian extension or any extensions from the school that would have blocked the chromebook. I'm still not sure why the chromebook wasn't administered but I don't have the chromebook anymore. I also got the chromebook from the school's library where you would usually ask the librarian for another school chromebook if you had lost your previous one or it was broke. So I ask how do you unbrick the chromebook? Because even when you do "esc + reload tab button + power button" it does absolutely nothing. Also doing "ctrl + shift + alt + r" it also does nothing. So please, if anyone has a way to do so, please reply.

If there wasn't one then there would be know reason for the word "unbrick" :moyai:

**Comment by amelia-bartlett** [comment]
> I ask this because on another chromebook that I had was not controlled or was not managed by the school admin and so, there was no goguardian extension or any extensions from the school that would have blocked the chromebook. I'm still not sure why the chromebook wasn't administered but I don't have the chromebook anymore. I also got the chromebook from the school's library where you would usually ask the librarian for another school chromebook if you had lost your previous one or it was broke. So I ask how do you unbrick the chromebook? Because even when you do "esc + reload tab button + power button" it does absolutely nothing. Also doing "ctrl + shift + alt + r" it also does nothing. So please, if anyone has a way to do so, please reply.

did you try connecting a power source

**Reply by amelia-bartlett** [reply]
it legit sounds like it either completely ran out of power (except maybe the cmos battery if it is separate), has corrupt firmware, or damaged hardware

also if you have permission to open it make sure the battery and display are both properly plugged into the mainboard.
